optaplanner-website-root/content/compatibility/kubernetes-openshift.adoc

```diff
@@ -0,0 +1,11 @@
+= Artificial Intelligence solver for Kubernetes and OpenShift
+:jbake-type: compatibilityBase
+:jbake-description: Use OptaPlanner (open source) for Artificial Intelligence planning optimization on Kubernetes and OpenShift.
+:jbake-priority: 1.0
+:jbake-related_tag: kubernetes
+:showtitle:
+
+OptaPlanner runs great on Kubernetes,
+especially on https://www.redhat.com/en/technologies/cloud-computing/openshift[OpenShift].
+
+Start a https://developers.redhat.com/developer-sandbox[Developer Sandbox for OpenShift] to deploy one of the https://github.com/kiegroup/optaplanner-quickstarts[OptaPlanner Quickstarts] to it.
```

Comment:
How straightforward it actually is? Unless the experience is good and we have step-by-step instructions in the READMEs, I would be very careful about nudging users to try it. Once they get a bad experience, they won't return.

Reply:
It will be straightforward soon. I have some local work, we need to add .s2i dirs IIRC.
Yes, we 'll need to add step-by-step instructions in the readme

Reply:
I'll try to get the other PR's that add that .s2i dirs later this week so we can merge it all together.

Reply:
Will be fixed by https://github.com/kiegroup/optaplanner-quickstarts/pull/272